hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -329,10 +341,12 @@ private function up1402_removeUnusedCaseTypes() {
    * Creates/Updates default case types to be shipped
    * with this extension
    *
+   * WARNING: Resets activity types for an assignment, discarding customization
```

Comment:
I think by adding this flag doesn't match what's requested for this ticket. Mike told me that since assignments are not widely customized yet it's fine to override them. By adding a flag to not update them if they exist I think it would fail to pass testing (documents would not be added). 

Since all case types will already be added by previous upgrades then calling this method with `true` would have no effect.